receiver/windowseventlogreceiver/receiver_windows_test.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/consumerretry"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/adapter"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/input/windows" 
```

Comment:
?

Reply:
This and the above change were just the results of `make gofmt` - I had concerns that they would cause CI to fail if I omitted them.

Reply:
They shouldn't be needed. They can be changed in another PR if you want. (e.g. run `make -C receiver/windowseventlogreceiver fmt` and PR that)

Reply:
Yanked them out and started https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21975